### DISCUSSION TITLE
Need help to extract this table example

### DISCUSSION THREAD

**Discussion by whoiswentz** [jsvine/pdfplumber#1146] [Ask for help with specific PDFs]
I have attached the sample pdf.

I need help extracting this sample table into a pandas data frame, I need the keys, values, and letters, I tried to use explicit lines but It didn't work

[sample.pdf](https://github.com/user-attachments/files/15506392/sample.pdf)

**Comment by jsvine** [comment]
Hi @whoiswentz, it seems that some of the difficulty is coming from the fact that there aren't many graphical indicators (rather than simple positioning of the letters) of the separation between the columns. Using the `"explicit_vertical_lines": [...]` setting along with `"horizontal_strategy": "text"` might work here. Can you share a reproducible code example of where explicit lines didn't seem to be working for you?
